internal/pkg/config/config.go

```diff
@@ -47,15 +52,25 @@ const (
 	SPOdImagePullPolicy = "SPOdImagePullPolicy"
 )
 
+var ErrPodNamespaceEnvNotFound = errors.New("the env variable MY_POD_NAMESPACE hasn't been set")
+
 // GetOperatorNamespace gets the namespace that the operator is currently running on.
-func GetOperatorNamespace() string {
-	// TODO(jaosorior): Get a method to return the current operator
-	// namespace.
-	//
-	// operatorNs, err := k8sutil.GetOperatorNamespace()
-	// if err != nil {
-	// 	return "security-profiles-operator"
-	// }
-	// return operatorNs
-	return "security-profiles-operator"
+func GetOperatorNamespace() (string, error) {
+	// This is MY_POD_NAMESPACE should have been set by the downward API to identify
+	// the namespace which this controller is running from
+	MyPodNamespace := GetEnvDefault("MY_POD_NAMESPACE", "")
```

Comment:
How about:
```suggestion
	MyPodNamespace := util.EnvDefault("MY_POD_NAMESPACE", "")
```

And importing util from `k8s.io/release/pkg/util`